garden-backend-service/src/api/schemas/modal/modal_function.py

```diff
@@ -12,6 +12,7 @@ class ModalFunctionMetadata(CommonFunctionMetadata):
     doi: str | None = None
     # modal supports conda requirements but entrypoints don't
     conda_requirements: list[str] = Field(default_factory=list)
+    example_usage: str = ""
```

Comment:
I wasn't thinking originally about retaining both the original raw text we parsed out and the "cleaned up" text the user confirms. But this makes a lot of sense, nice